src/components/QuestionPlay/WithoutOptions.tsx

```diff
@@ -18,14 +20,50 @@ export default function WithoutOptions({
   options,
   isAttempted,
   setIsAnswerRevealed,
-  setSelectedChoices,
+  submitResponse,
+  continueGame,
 }: Props) {
-  function handleSelectedChoice(choice: number[]) {
-    setSelectedChoices(choice);
+  const [isUndoTimerRunning, setIsUndoTimerRunning] = useState(false);
+  const [time, setTime] = useState(0);
+  const [isUndoClicked, setIsUndoClicked] = useState(false);
+  const timerRef = useRef(null as any);
+  const userResponse = useRef(null as any);
 
-    setTimeout(() => {
-      document.getElementById('btnSubmitResponse')?.click();
-    }, 100);
+  useEffect(() => {
+    if (isUndoTimerRunning) {
+      timerRef.current = setTimeout(() => {
+        if (time === 4) {
+          handleContinueClick(userResponse.current);
+        }
+
+        setTime(time + 1);
```

Comment:
We don't need to do this when time === 4? Maybe put this in else?

Reply:
fixed